syntax = "proto3";

service ChatGPT {
    rpc ChatWithGPT(ChatRequest) returns (ChatResponse) {}
}

message ChatRequest {
    string prompt = 1;
}

message ChatResponse {
    string response = 1;
}
